apps/registry/scripts/build-registry.ts

```
@@ -2,9 +2,31 @@ import { promises as fs, readFileSync } from "node:fs";
+  return {
+    key: key.replace(/^\./, ""),
+    value: Object.keys(value)
+      .map((key) => key.replace(/^\@apply /, "").replaceAll("-aui-", "-"))
+      .filter((k) => k)
+      .join(" "),
+  };
+});
+
+const transformClassnames = (content: string) => {
+  for (const style of stylesToReplace) {
+    content = content.replaceAll(
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)